splinterd/tests/framework/network.rs

```diff
@@ -91,9 +127,37 @@ impl Network {
                 .with_admin_signer(admin_signer)
                 .with_signers(signers)
                 .with_external_registries(self.external_registries.clone())
-                .with_biome_enabled()
-                .build()?
-                .run()?;
+                .with_biome_enabled();
+            let builder = match (self.biome_auth, self.cylinder_auth) {
```

Comment:
Why the match statement? Couldn't you just do

```rust
if self.biome_auth {
   ...
   builder = builder.with_biome_auth(...);
}

if self.cylinder_auth {
    builder = builder.with_cylinder_auth(...);
}
```

Seems clearer to me, given there's no relationship between the two auth methods.

Reply:
fixed